CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Zonemaster::Backend
+
+Contribution to this repository is welcome. Contribution can be either an issue
+report or a code or a documentation update. Also see the information in the
+[main README][Zonemaster/Zonemaster README] in the main Zonemaster respository.
+
+## Issue
+
+First search for a similar issue in the [issues list]. If a relevant issue is
+found, add your information as a comment. If no relevant issue is found, create a
+[new][create issue] issue. Give as many details as you have and describe, if
+possible, how the issue can be reproduced.
+
+## Pull request
+
+If you have a solution for an issue, or a suggestion for an improvment, please
+first search for a pull request for the same issue among open [pull requests].
```

Comment:
There are lots of issues and pull requests in this sentence. I suggest this wording instead:

> If you'd like to contribute a change, first please look for issues and open pull requests that are about the same thing.